### DISCUSSION TITLE
[BUG]5p can't find the block！！！！

### DISCUSSION THREAD

**Discussion by asgadolph** [Chia-Network/chia-blockchain#8168] [Support]
I have 5p plots; The block has not been found for more than two consecutive times and seven days this month. All settings are OK and the network is unobstructed. I don't know whether it's me or everyone.

I hope to get a reply and help solve it. Thank you very much!

bter888@163.com

**Comment by markusmazurczak** [comment]
Are you pool mining or solo?
What are 5p plots? Do you have 5 plots?
What are "two consecutive times"?

Why are you that sure that you should have won during "your" consecutive times?

**Comment by max4711qq** [comment]
There are two threads about this. You are in good company unfortunately

https://github.com/Chia-Network/chia-blockchain/discussions/8074

https://github.com/Chia-Network/chia-blockchain/discussions/7939

Chia network is not working as it is supposed to be. Unfortunately the small "farmers" are not aware of this because their time to win is beyond ...

**Comment by asgadolph** [comment]
![Uploading 微信图片_20210823035058.png…]()
![微信图片_20210823035050](https://user-images.githubusercontent.com/84242772/130368199-e8d26091-855f-4ce4-b56f-fdb4cf099862.png)

**Comment by asgadolph** [comment]
![微信图片_20210823035058](https://user-images.githubusercontent.com/84242772/130368220-ac0cee96-5847-48bf-b495-613e052e58bd.png)

**Comment by asgadolph** [comment]
30000og plots burst abnormally! Up to now, the block has been discovered once in 7 days. And 16000 new plots have not found blocks.

**Comment by asgadolph** [comment]
There is a problem with the time when the block was found.

**Comment by emlowe** [comment]
The blockchain is progressing, blocks are being created, and block rewards are being distributed as designed. Just because you aren't farming a block doesn't mean something is wrong. Every block farmed was in fact farmed by someone - which means that they found a valid proof. There are no favourites played in the math. There are plenty of large farmers who on occasion go through stretches where wins seem less often. Check your logs, make sure you lookups are fast and make sure your internet connection is good. You have 28 seconds to get your proof to a timelord.

This sounds like a support question and not a bug report

**Comment by max4711qq** [comment]
as per
#8074
#7939
it's stated that log files are OK, lookups are fast enough and that the so often used word "luck" is not appropriate. (>10 times of estimated time to win)

**Comment by tomcat6** [comment]
I think the cause is timelord。
would you like to show you log? 
command as follow:
 cat ~/.chia/mainnet/log/debug.log |  grep "Found [1-9] proofs"

**Reply by max4711qq** [reply]
I didn't get any proofs. (Checked my logs for >30 days with chiadog and manually). But I got those proofs before from time to time and then one again after >10 times ETW. Then nothing again for >6 x ETW. After this I wiped all my ca. 4PB of data clean and decided to not waste resources on this anymore.

**Comment by psun888** [comment]
It's not your issue at all. It's Chia with flawed designs. Unfortunately, doesn't matter you, or any farmer, paid a fortune to support Chia network, the programmers of Chia software don't care as far as they have made fortune on top of your suffers. It's a common issue, just bear with it for now mate. Maybe, someday a better version of a new storage based coin will rise to the love.

**Comment by emlowe** [comment]
Your proof must get to a timelord within 28 seconds. Remember the chain is not stalled, so lots of farmers are having no trouble getting rewards. Please keep in mind the first word in ETW is "estimated". It's not like you are guaranteed to get rewards within that period of time. If you want smoother rewards, the general recommendation is to join a pool

**Reply by alexwwstx** [reply]
if it happens pretty often for some big farms with > 1PB plots (which is TRUE based on discussion here), then I will say something is most likely not right if it's just a pure probability math.

On the other side, there's a discussion in chia forum showing farmer(s) can win several times within very short period by < 100T plots. Pure probability math cannot work again.

**Comment by max4711qq** [comment]
Sure. but I am a maths guys. >10x ETW is not just "unlucky" if it happens several times. (To several guys I know)

**Comment by markusmazurczak** [comment]
I am interested in the total amount of people complaining that they have very very big ETW's.
There are already 10,000s of people farming chia and as far as I followed the tickets here there are, lets get big, 50 people complaining about this.
If you do the math here again you will see that this is such a low percentage that the possibility of big bad luck becoms more likely.

Dont get me wrong. I dont want to nip the issue in the bud but I dont really think that there is a problem regarding impossibility to find blocks and get them confirmed. If this is the case there wont be that much people farming I guess

**Reply by max4711qq** [reply]
people tend to go to a forum when they have issues. Not when things go well for them.

**Reply by markusmazurczak** [reply]
Exactly. But this does not disprove my assumption that only 0.1% of people have the described problem. Much more it reinforces it.

**Reply by max4711qq** [reply]
Majority of people have not enough plots to find out that they are not winning what they should. So the 0.1% cannot be correct.

**Reply by markusmazurczak** [reply]
The majority of people are pool mining, I guess. So competing with OG plots against pools make that even more "difficult" to win solo...

But, lets take for example "SpacePool". The current leader (in case of plots) has 27PiB of plots which gives him around 1% of round share. He is mining for 41 days in that pool and has found 81 blocks.

This is in my opinion an example that it works as expected.

**Comment by max4711qq** [comment]
now you are mixing things. This makes no sense. No matter if solo or pool, if ETW of >10 happens regularly, something is not right. 
No matter what the pool guys do, but here are several serious examples of farms >1 PB being more unlucky than statistically possible. And I know more people with the same issues. Beyond this forum. But it doesn’t matter. Some people are aware of this now and had the change to discuss their problems; while it seems to work fine for others. I am just saying that it’s not true that only 0.1% of farmers have issues.

**Comment by psun888** [comment]
Seems a smart guy, but logically so wrong. Billions of people use Windows, however only a few people reporting bugs, are those people something wrong. Chia is very bad designed software with hundreds, if not thousands, problems. One is the scale problem. Sure, big pools seems OK, but the middle sized miners have problems. I just wonder why you attack users who report problems?!! weird.

**Reply by markusmazurczak** [reply]
So show me where I am attacking someone. I am just having the point of view that there is no general problem like the one described and I am discussing this.

But if you prefer to see yourself in the victim role of the one who is always wronged by everyone, that's your problem, but leave me alone with your kid stuff. Sorry but I am so sick of people like you who think they are beeing attacked when going into somewhat of discussion.

Anyway ... You are comparing an Bug in Microsoft with an statistically induced problem. I think that this is not right. If there are billions of people having all the same issue like the thousands that reported it, why are still billions of people using the software?

Again, dont get me wrong. I understand your issue from an emotional standpoint but whe are talking about statistics and a handfull of people having the described issue.

**Comment by max4711qq** [comment]
the thing with "only a handfull" is not correct. Most of the farmers will just never find out because of their small operation. They are waiting for their x months ETW and being told the thing is a lottery. Time will tell. Good luck to everybody.

**Reply by markusmazurczak** [reply]
But this is exactly what I am asking for. When it is "not a handfull". How many farmers do you know. Dont get me wrong. Trying to get into real numbers makes it more practical to talk about wrong statistics.

You stated that you know "more people" with the same issue. Where are all this people (I am not active in ary forum or other chat like stuff)?
Do you know any official comment regarding this issue from the devs. I am rly interested in this topic.

As an couterexample, before the official pooling protocol goes online and only after 1 month of plotting and farming I won a block with only 40TB of plots

**Comment by max4711qq** [comment]
I've sold 2nd hand hardware to (mostly) CHIA "farmers" worth 4 million USD between May and July. This is why I know many guys with serious investments and pretty large "farms".

The counterexample is nice (congrats!) but proofs nothing. A quick win with a small setup seems to be pretty common. I also had this in several test setups with 100TB and less. 

BUT - this is the thing:
knowing 10+ operations (including some guys in this forum) - all of them running into a LESS THAN ONE PERCENT chance of being "unlucky" - proofs that something is not right. Maybe I should start a lottery team together with these guys. (For example: my own experience, 3PB and only one block in 25 days is a 0.046% chance to happen). 

How many guys do you know who regularly win a block, more or less in line with ETW? I don't know anybody. 

This is my last answer/comment on all of this because I am not invested in this anymore and this is just going ping pong. There will not be a clear proof or winner in this anyway. But it was good to have this discussion. I am sure others will contribute more to this - hopefully successful guys as well, telling everybody else how well it goes for them. Chia is a pretty young project and I am sure that this "issue/topic" will get official attention as well in the near future. 
Good luck with your operation.

**Comment by BrandtH22** [comment]
@asgadolph , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
